keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -964,7 +973,7 @@ operations covered by [existing SLIs/SLOs]?**
 
 * **Will enabling / using this feature result in non-negligible increase of 
 resource usage (CPU, RAM, disk, IO, ...) in any components?**
-  No - Privileged containers will honor limits/reserves specified in the specs and will count against node quota just like unprivilged containers.
+  No - `HostProcess` containers will honor limits/reserves specified in the specs and will count against node quota just like unprivileged containers.
```

Comment:
If the host process container starts other processes on the node will those processes be constrained to the same job object?

Reply:
They will be

Reply:
https://docs.microsoft.com/en-us/windows/win32/procthread/nested-jobs